### PR TITLE
Remove old scratch code

### PR DESCRIPTION
**Pull request by dmcavoy**
Before Sprite Lab, Scratch was considered as another option for replacing Play Lab and giving younger students more ability to make diverse projects. JoshL made a prototype of pulling scratch into our website. No one is using this to create content so I am recommending we remove it. 

There were only 2 Scratch levels. One for testing. The other the base level for a new project to be created off of. The levels were not in any scripts.

### CODE REVIEW

**Comment by maddiedierker** [comment]
thank you for this, dani! 

looking at test failures, there are some Spritelab UI test failures that look related to this change. it looks like you should be able to go to `/projects/spritelab/new` to reproduce the error (that's where the UI tests are failing)

also cc'ing *labs (@epeach @jmkulwik @ajpal) just in case any of them have context around scratch

File: dashboard/app/models/game.rb [hunk 1]
```diff
@@ -301,7 +296,6 @@ def channel_backed?
     CurriculumReference:curriculum_reference
     Map:map
     CustomFlappy:flappy
-    Scratch:scratch
```
Comment (ajpal) [inline dashboard/app/models/game.rb]: The order of this list matters, so removing this entry will probably break dance, spritelab, bubblechoice, and fish.

Comment (dmcavoy) [reply dashboard/app/models/game.rb]: Whats the best way to deal with this then? Then leave the entry?

Comment (ajpal) [reply dashboard/app/models/game.rb]: Yeah probably just leave it and add a comment

**Comment by dmcavoy** [comment]
@madelynkasula @ajpal The change is now passing tests. Can you re-review?

**Review by maddiedierker — APPROVED** [review]
thank you for doing this, dani!

**Pull request merged**